src/Web/Sfa.Das.Sas.Web/Views/Apprenticeship/Standard.cshtml

```diff
@@ -49,9 +49,59 @@
                     </ul>
                 </div>
             </dd>
+      
+       
         </dl>
+        <div class="grid-row"> 
+            <div class="column-full">
+                <p class="collapsible">End Point Assessment </p>
+            @if (Model.AssessmentOrganisations.Any())
```

Comment:
` @if (Model.AssessmentOrganisations.Any())` should handle null exception

Reply:
sounds good, I'll address